features/api/library_creation_requests.feature

```diff
@@ -19,7 +19,7 @@ And I have a "full" authorised user with the key "cucumber"
 
```

Comment:
So what follows is a lot of fixes for tests. Unfortunately the added uniqueness validation broke things, as the tests kept re-creating the 'Standard' library type.
The fixes entailed:
- Don't automatically associate library_type and library type validators with requests
- Explicitly do so when we need to test them